frontend/src/Component/EditBikeForm/EditBikeForm.tsx

```diff
@@ -56,6 +57,7 @@ export default function AddBikeForm() {
                         required
                         id="outlined-required"
                         label="Mileage"
+                        value={mileage}
                         error={!/^\d+$/.test(mileageFieldValue.trim()) && mileageFieldValue!==""}
                         helperText={(!/^\d+$/.test(mileageFieldValue.trim()) && mileageFieldValue!=="")
```

Comment:
Apparently thats a bad idea, because there are some problems with type = "number". I do inputProps={{ inputMode: 'numeric', pattern: '[0-9]*' }} instead. It also depends on the browser for working. Her is an article about it: https://stackoverflow.blog/2022/12/26/why-the-number-input-is-the-worst-input/